# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/net-tools/hotfix-sbin.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- net-tools-2.10/Makefile.vanilla	2025-07-29 16:19:58.157008299 +0200
+++ net-tools-2.10/Makefile	2025-07-29 16:20:35.376603203 +0200
@@ -206,15 +206,14 @@
 installbin:
 	@echo
 	@echo "######################################################"
-	@echo "Notice: ifconfig and route are now installed into /bin"
 	@echo "######################################################"
 	@echo
 	install -m 0755 -d ${BASEDIR}${SBINDIR}
 	install -m 0755 -d ${BASEDIR}${BINDIR}
-	install -m 0755 ifconfig   ${BASEDIR}${BINDIR}
+	install -m 0755 ifconfig   ${BASEDIR}${SBINDIR}
 	install -m 0755 nameif     ${BASEDIR}${SBINDIR}
 	install -m 0755 netstat    ${BASEDIR}${BINDIR}
-	install -m 0755 route      ${BASEDIR}${BINDIR}
+	install -m 0755 route      ${BASEDIR}${SBINDIR}
 ifeq ($(HAVE_ARP_TOOLS),1)
 	install -m 0755 arp        ${BASEDIR}${SBINDIR}
 	install -m 0755 rarp       ${BASEDIR}${SBINDIR}
